src/SUMMARY.md

```diff
@@ -10,7 +10,7 @@
 * [Starter Templates](getting-started/starter-templates.md)
 * [Examples](getting-started/examples.md)
 
-## Core Concepts <a id="concepts"></a>
+## Core Concepts
```

Comment:
```suggestion
## Core Concepts <a id="concepts"></a>
```

We should keep this, otherwise Gitbook will try to migrate all the files from `concepts/` to `core-concepts/`

Reply:
Oops! Excuse me for my obsessive-compulsive disorder.

Reply:
Haha no worries I'm the same, I understand :)